pallets/loans/src/functions.rs

```diff
@@ -375,6 +376,11 @@ impl<T: Config> Pallet<T> {
 				// ensure amount is not more than current debt
 				let repay_amount = amount.min(debt);
 
+				let new_repaid_amount = loan_info
+					.repaid_amount
+					.checked_add(&repay_amount)
+					.ok_or(DispatchError::Arithmetic(ArithmeticError::Overflow))?;
```

Comment:
I think a `(ArithmeticError::Overflow)?;` is sufficient. But I don`t really care. No need to change this,

Reply:
I think we should change it, otherwise our case base is twice as big unnecessarily :)

Reply:
Ha, I changed these in the interest accrual PR already ;)